code/_helpers/unsorted.dm

```diff
@@ -1123,7 +1123,7 @@ var/list/wall_items = typecacheof(list(
 // Checks if user can use this object. Set use_flags to customize what checks are done.
 // Returns 0 if they can use it, a value representing why they can't if not.
 // Flags are in `code/__defines/misc.dm`
-/atom/proc/use_check(mob/user, use_flags = 0, show_messages = FALSE)
+/atom/proc/use_check(mob/user, use_flags = 0, show_messages = TRUE)

```

Comment:
This is a bad idea. The proc is called `use_check`, not `use_check_and_message`, meaning that the side-effect of showing a message is going to catch people off-guard. A lot of code presumes that `use_check` shows no message, since the error is handled internally or with a custom error message.

Please leave this as `FALSE` and create a new proc which is something like `use_check_and_message`. One such proc might already exist, if I understood Lohikar correctly.